### DISCUSSION TITLE
Change Solver Settings via R

### DISCUSSION THREAD

**Discussion by Erik-Sjogren** [Open-Systems-Pharmacology/Forum#1443] [Questions & Problems] [answered]
Hi all,

Is there a pre-defined function in the R package ospsuite to change solver settings?
Similarly to the one for changing the output interval [setOutputInterval()].

If not, I would be very thankful for some guidance (if possible) on how to accomplish this by other means.

Cheers,
Erik

**Comment by Yuri05** [comment, marked as answer]
you can access solver settings via the `$solver` property of a loaded simulation.
e.g.
<img width="350" src="https://user-images.githubusercontent.com/25061876/214292165-0e159079-e48e-45a2-a3db-add3b890f021.png">

s. also https://www.open-systems-pharmacology.org/OSPSuite-R/reference/SolverSettings.html

**Comment by Erik-Sjogren** [comment]
Splendid!
Many thanks
Erik
